testing/chain.go

```diff
@@ -404,7 +405,9 @@ func (chain *TestChain) ConstructUpdateTMClientHeaderWithTrustedHeight(counterpa
 	header := counterparty.LastHeader
 	// Relayer must query for LatestHeight on client to get TrustedHeight if the trusted height is not set
 	if trustedHeight.IsZero() {
-		trustedHeight = chain.GetClientState(clientID).GetLatestHeight().(clienttypes.Height)
+		clientState := chain.GetClientState(clientID)
+		clientStore := chain.GetSimApp().GetIBCKeeper().ClientKeeper.ClientStore(chain.GetContext(), clientID)
```

Comment:
maybe adding a `chain.GetClientStore()` function to the `TestChain` struct might clean up some of the test code a bit